plugins/linstor.yaml

```diff
@@ -3,57 +3,72 @@ kind: Plugin
 metadata:
   name: linstor
 spec:
-  version: v0.1.2
+  version: v0.2.7
   homepage: https://github.com/piraeusdatastore/kubectl-linstor
   shortDescription: View and manage LINSTOR storage resources
   description: |
-    LINSTOR is an open-source software designed to manage block storage devices for large Linux server clusters.
+    Usage:
```

Comment:
Usage strings are typically unnecessary and we print instructions when the plugin is installed.